css/properties/text-size-adjust.json

```diff
@@ -22,13 +22,9 @@
               "version_added": "54"
             },
             "edge": [
```

Comment:
I think [Travis is complaining](https://travis-ci.org/mdn/browser-compat-data/builds/355022139#L3247) because this should no longer be an array. Just `{"prefix":...,"version_added":...}`.